test/unit/connection_test.rb

```diff
@@ -26,6 +26,25 @@ def test_connection_endpoint_raises_uri_error
     end
   end
 
+  def test_connection_passes_env_proxy_by_default
+    @connection.logger.expects(:info).twice
+
+    spy = Net::HTTP.new('example.com', 443)
+    Net::HTTP.expects(:new).with('example.com', 443, :ENV, nil).returns(spy)
+    spy.expects(:get).with('/tx.php', {}).returns(@ok)
+    response = @connection.request(:get, nil, {})
+  end
+
+  def test_connection_does_pass_requested_proxy
+    @connection.logger.expects(:info).twice
```

Comment:
This seems unrelated to what we're testing here, can we remove it? Same question for the other test.